presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -796,6 +797,19 @@ public FeaturesConfig setDynamicFilteringRefreshInterval(Duration dynamicFilteri
         return this;
     }
 
+    public int getDynamicFilteringPerDriverMinMaxLimit()
+    {
+        return dynamicFilteringPerDriverMinMaxLimit;
+    }
+
+    @Config("dynamic-filtering-per-driver-min-max-limit")
```

Comment:
rename to `dynamic-filtering-min-max-limit-per-driver`

Reply:
renamed